schemas/app/PriceList.json

```diff
@@ -10,36 +10,28 @@
       "required": true
     },
     {
-      "fieldname": "enabled",
-      "label": "Enabled",
+      "fieldname": "isEnabled",
+      "label": "Is Price List Enabled",
       "fieldtype": "Check",
       "default": true
     },
     {
-      "fieldname": "buying",
-      "label": "Buying",
+      "fieldname": "isSales",
+      "label": "For Sales",
       "fieldtype": "Check",
-      "default": false,
-      "required": true
-    },
-    {
-      "fieldname": "selling",
-      "label": "Selling",
-      "fieldtype": "Check",
-      "default": false,
-      "required": true
+      "default": true
```

Comment:
why `isSales` default value set to `true` ?

Reply:
Seems like a better default. Users make Sales more often than Purchases.

Reply:
okay got it